themes/OpenFisca-2022/layouts/partials/card.html

```diff
@@ -25,5 +28,6 @@ <h3 class="card_title">{{ .title }}</h3>
 			</ul>
 		</div>
 	{{ end }}
-	<a class="card_link" target="_blank" href="{{ .website | safeURL }}">{{ .website | safeURL }}</a>
+	{{ if .until }}<p class="text-small"><span class="text-grayness">{{ .until | time.Format "January 2006" }} - </span><a href="{{ .website | safeURL }}" class="link-isDisabled" rel="noFollow">{{ .website }}</a></p>{{ end }}
```

Comment:
If you disable the link (which makes sense since we expect URL to not exist anymore, why mark it as an `<a>` and add a `isDisabled` class to counter this? 🤔 Wouldn't it be simpler to just use `<span>`?